styles/style.css

```diff
@@ -232,3 +261,28 @@ height: 100%;
     background: var(--color-accent);
 }
 
+
+.promotion-box {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1920px;
+    background-color: #F2F0FF;
+}
+.promotion-box .container {
+    display: flex;
+    align-items: center;
+    gap: 22px;
+}
+.promotion-box .container .text-box {
+ display: flex;
+ flex-direction: column; 
+ max-width: 644px; 
+}
+.promotion-box .container .text-box p:nth-child(1) {
+color:var(--color-accent);
+line-height: 28px;
+font-size: 16px;
+font-family: Arial, "Lato", sans-serif;
+}
```

Comment:
Delete it. You work only with navigation. This section should be done at other branch that you will create after merging this one